code-challenges.js

```diff
@@ -17,8 +64,14 @@ var arrayOfWords = ["Apple", "Banana", "Plum", "Cherry", "Kiwi", "Peach"]
 // Expected output: "Apple" "Banana" "Peach"
 
 
-
-
+// Start by creating a function that takes in an array and returns words containing letter a
+const letterA = (array) =>{
+    // create a local variable and start filtering our array and using .includes in order to isolate words that include the letter "a"
+    newArr = array.filter(value => value.includes("a") || value.includes("A"))
```

Comment:
Consider a more dynamic way of checking for a letter without having to hard code the upper and lower case.